### PR TITLE
[PBI-1847]

### PR DESCRIPTION
**Pull request by dchauOoyala**
- Add player configuration parameter: "cacheBuster". Default true. Only
an explicit value of "false" will turn off cacheBuster functionality
- Add function to replace CACHEBUSTING macro in tracking urls before pinging

### CODE REVIEW

File: js/ssai_pulse.js [hunk 1]
```diff
@@ -899,6 +922,26 @@ OO.Ads.manager(function(_, $)
...
+     * @private
+     * @method SsaiPulse#_cacheBuster
+     * @param {string[]} urls The array of URLs
+     * @returns {string[]} The new array of URLs.
+     */
+    var _cacheBuster = _.bind(function(urls)
+    {
+      for (var i = 0; i < urls.length; i++)
+      {
+        var regex = /%5bcachebusting%5d|\[cachebusting\]/i;
```
Comment (aeng7) [inline js/ssai_pulse.js]: Can we try decodeURIComponent instead of %5b and %5d?

Comment (aeng7) [inline js/ssai_pulse.js]: Can we also store the [cachebusting] value in a var?

File: js/ssai_pulse.js [hunk 2]
```diff
@@ -138,6 +141,14 @@ OO.Ads.manager(function(_, $)
     this.loadMetadata = function(adManagerMetadata, backlotBaseMetadata, movieMetadata)
     {
       this.ready = true;
+
+      if (adManagerMetadata)
+      {
+        if (adManagerMetadata["cacheBuster"] === "false")
```
Comment (pilievOoyala) [inline js/ssai_pulse.js]: in case we change the default, can you add in reading if === "true"

**Comment by pilievOoyala** [comment]
LGTM

**Pull request merged**